be/src/olap/lru_cache.h

```diff
@@ -146,6 +147,7 @@ class CacheKey {
 enum class CachePriority { NORMAL = 0, DURABLE = 1 };
 
 using CacheValuePredicate = std::function<bool(const void*)>;
+using CacheValueExtractor = std::function<int64_t(const void*)>;
```

Comment:
add some comment

Reply:
Is CacheTimeExtractor more intuitive?

Reply:
However, it is better to name it with time first